backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -179,12 +180,47 @@ export class ContributionResolver {
   async listAllContributions(
     @Args()
     { currentPage = 1, pageSize = 5, order = Order.DESC }: Paginated,
+    @Arg('filterState', () => [ContributionStatus], { nullable: true })
```

Comment:
On the left hand side you have State and the right hand side is Status. I prefer Status.

Reply:
Change everything to statusFilter